benchmarks/spark/emr.tf

```
@@ -0,0 +1,250 @@
+provider "aws" {
+  region = "us-east-2"
+}
+
+resource "aws_s3_bucket" "emr_bucket" {
+  bucket        = "spark-benchmark-python-script-bucket-${random_id.bucket_id.hex}"
+  force_destroy = true
+}
+
+resource "random_id" "bucket_id" {
+  byte_length = 8
+}
+
+resource "aws_s3_object" "python_script" {
+  bucket = aws_s3_bucket.emr_bucket.id
+  key    = "scripts/spark_nyc_taxi_preciptation.py"
```

Can we change the name of this file in this PR since I believe there was a typo originally? 
```suggestion
  key    = "scripts/spark_nyc_taxi_precipitation.py"
```
And change the file name too

---

Good catch, fixed